### DISCUSSION TITLE
Query Monitoring

### DISCUSSION THREAD

**Discussion by golu360** [supabase/supabase#12394] [Feature Requests]
In built query monitoring tool based off [pg_stat_statements](https://www.postgresql.org/docs/current/pgstatstatements.html)

- Can be an opt in feature
- DB server needs a restart post opting in for pg_stat_statements.

**Comment by pranavbadami** [comment]
@golu360 I believe this has already been added in: https://supabase.com/docs/guides/platform/performance
